spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/servlet/error/DefaultErrorAttributes.java

```diff
@@ -94,7 +102,7 @@ private void storeErrorAttributes(HttpServletRequest request, Exception ex) {
 	private void addStatus(Map<String, Object> errorAttributes,
 			RequestAttributes requestAttributes) {
 		Integer status = getAttribute(requestAttributes,
-				"javax.servlet.error.status_code");
+				RequestDispatcher.ERROR_STATUS_CODE);
```

Comment:
We have a few places where we've avoiding referencing `RequestDispatcher` directly to help with Servlet 2.5 compatibility. My preference would be to take care of all of them in a single, separate change.